multi-node/aws/pkg/config/config.go

```diff
@@ -413,10 +504,18 @@ func getContextString(buf []byte, offset, lineCount int) string {
 	return string(buf[leftLimit:rightLimit])
 }
 
+type etcdInstance struct {
+	IPAddress   net.IP
+	SubnetIndex int
```

Comment:
Where is this indexed? The config file? There needs to be an explicit "order cannot ever change in this config file" if that's the case.